### ISSUE TITLE
Issue with accuracy in ShapeNet dataset

### ISSUE THREAD

**Issue by edshkim98** [XuyangBai/KPConv.pytorch#6]
Hi @XuyangBai , I am currently trying your code for part-segmentation.
I am currently on epoch 250, but it seems like the IOU is still quite low.
In the paper, the IOU for chair in shapenet got over 90 IOU, but I am getting somewhere under 50 IOU...
Is there any solution to increase the accuracy similar to the original KPconv paper?
Can you also tell me what your IOU was when you were training shapenet part segmentation?
Thank you for your help in advance.

**Comment by XuyangBai** [comment]
Hi @edshkim98 Thanks for your interest in my implementation. But I implemented this version very long time ago and have no time exploring the performance now. Would you mind trying the official PyTorch version of KPConv https://github.com/HuguesTHOMAS/KPConv-PyTorch/?
